benchmarking/definitions/definition_resnet_cifar10.py

```diff
@@ -12,8 +12,8 @@
 # permissions and limitations under the License.
 from pathlib import Path
 
-from benchmarks.utils import get_cost_model_for_batch_size
-from examples.training_scripts.resnet_cifar10.resnet_cifar10 import \
+from benchmarking.cli.utils import get_cost_model_for_batch_size
```

Comment:
Could this rather be in benchmarking.utils? It does not pertain to cli.

Reply:
fixed